src/ZeroCrossImpl.java

```diff
@@ -0,0 +1,177 @@
+import java.util.Scanner;
+
+public class ZeroCrossImpl implements ZeroCross {
+
+    int whoPlays;
+    public int firstPlayer;
+    public int secondPlayer;
+
```

Comment:
You should use encapsulation principle.
Use "private" access and create methods to work with field.